shared/log.go

```diff
@@ -93,5 +93,5 @@ func LogCritf(format string, args ...interface{}) {
 func PrintStack() {
 	buf := make([]byte, 1<<16)
 	runtime.Stack(buf, true)
-	LogDebugf("%s", buf)
+	LogErrorf("%s", buf)
```

Comment:
Well, this is leftover from debugging. But it seems like we might want to do this anyway, since I have to do it every time I debug anything :(